tests/console/yast2_dns_server.pm

```diff
@@ -139,5 +139,23 @@ sub run {
     $self->assert_enabled(0);
 }
 
+sub post_fail_hook {
+    my $self = shift;
+
+    # package info
+    $self->save_and_upload_log('rpm -qi bind', '/tmp/rpm_info_bind.txt');
+    $self->save_and_upload_log('rpm -ql bind', '/tmp/rpm_listfiles_bind.txt');
+    $self->save_and_upload_log('rpm -qa',      '/tmp/rpm_all_packages.txt');
+    # iptable rules & ports
+    $self->save_and_upload_log('iptables -L -v',                '/tmp/iptable_ports.txt');
+    $self->save_and_upload_log('iptables -S -v --line-numbers', '/tmp/iptables_rules.txt');
+    $self->save_and_upload_log('firewall-cmd --list-services',  '/tmp/firewalld_services') if ($self->firewall eq 'firewalld');
+    # information logs from systemd & journald
+    $self->save_and_upload_log('systemctl --no-pager status named', '/tmp/systemctl_status_named.log');
+    $self->save_and_upload_log('journalctl -u named',               '/tmp/journactl_named.log');
+    # DNS configuration
+    upload_logs('/etc/named.conf');
```

Comment:
Do we want to call usual log collection (e.g. yast logs)? Then we can skip collecting packages list (as we have it already) and not filtered journalctl. WDYT?

Reply:
Calling the usual log collection routine is too generic and broad in case we need specific information about some service. Also some information is missing as iptables, firewalld or even configuration files for instances named.conf. I have just added missing y2logs.